akka-stream/src/main/scala/akka/stream/scaladsl/Restart.scala

```diff
@@ -464,7 +464,7 @@ private abstract class RestartWithBackoffLogic[S <: Shape](
         if (finishing || maxRestartsReached()) {
           fail(out, ex)
         } else {
-          log.error(ex, "Restarting graph due to failure")
+          log.warning("Restarting graph due to failure: {}", ex.getMessage)
```

Comment:
good, I think the logging should be more adjustable for the restart operator, but that is another issue

Reply:
I'd gladly make this more configurable in this change, but if you think its a separate concern, then I won't yet.

I can understand a general desire to make this more configurable, but I'm not too sure what specific variability clients want. My only desire is to vary the log level, and even that I don't really need variation, just reduction for all my current use cases.